docs/configuration.asciidoc

```diff
@@ -172,9 +172,9 @@ Use this setting in case the APM Server requires a token (e.g. APM Server in Ela
 | `http://localhost:8200` | List
 |============
 
-The URLs for your APM Servers. The URLs must be fully qualified, including protocol (`http` or `https`) and port. To add multiple servers, separate them with a comma (`,`).
```

Comment:
I'd keep the `To add multiple servers, separate them with a comma (`,`).` part - I think that adds value, even if the feature is not implemented.

Reply:
I'm not sure I follow your reasoning - you would like to let users know how to do something we don't support yet?

Reply:
Yes, otherwise it's not clear how to set multiple values. 

It's like we say:
- The setting enables you to add multiple servers.
- You add multiple values by separating by comma
- Currently only the 1. one is used.

So reading this short part makes it clear.

Otherwise we say:
- The setting enables you to add multiple values
- We use only the 1. one.
- (we don't even tell you how to set multiple values) 

But then why we say `The URLs for your APM Servers`?

Reply:
I see. So wouldn't fixing all other mentions of "URLs" in plural be a better approach?

Reply:
I'd be very open in the docs about this and not try to hide it. I personally would prefer the other one, because without the plural we still leave questions open (like why is the name of the setting itself in plural and how to set multiple values - even if only the 1. is used). But I'm also fine with this version. Maybe @bmorelli25 has an idea on this - I'd usually be super open in the docs and try to cover everything and just list the limitations.

Reply:
I agree that we should cover the limitations (which this PR already does). I just don't see the point of describing how to use a feature that is not implemented yet.

Reply:
>  I just don't see the point of describing how to use a feature that is not implemented yet.

To be fair, it's a partly implemented feature - we indeed parse the list, it's just that the `PayloadSenderV2` only uses the 1. item - that causes the trouble here.

I think it's just not good to diverge in the docs from the real implementation in order to hide something that we should not be ashamed of.

This setting is indeed a `List`, not a `String` - I don't see any problem with fully describing the feature and say that it only partly works in the current version instead of having docs that diverge from the real implementation (stating Type `String` when in reality it's a `List`) and leave questions open (why is the setting plural and how to set multiple values?).

From the users point of view I think an approach like this would be best (pseudo docs):
- The setting is `ServerUrls`, it's type is `List`
- Set multiple values by separating by comma
- The current version only uses the 1. one, feature will be implemented to use multiple in a later version.

Reply:
It doesn't matter to the end user if feature implemented 0% or 99% - in the end it either works or it doesn't. Setting this configuration option to a list doesn't work so there's no point in wasting user's time on reading how to configure more than one URL just to follow it up with a note saying that it's not yet supported.

Reply:
Ok, I give up on this.